### ISSUE TITLE
Blank Email from Sender Causes Error

### ISSUE THREAD

**Issue by Koshroy** [madeingnecca/haraka-plugins#2]
I have applications that are sending me diagnostic reports to root@domain, but are leaving the FROM field blank for bounce messages. The maildir plugin is attempting to split this empty email address. I can fix the bug if you feel like the maildir plugin should handle empty FROM fields. On the other hand, maybe you don't feel like the maildir plugin should have to deal with an empty FROM field, and a previous plugin should take care of it. Let me know what you think.

**Comment by madeingnecca** [comment]
Yes, I think the plugin should not raise errors in case of missing "mail from" address.
Feel free to fix it if you want. Thanks.
